ui/app/i18n/ipd/locale_pt_BR.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
translations should not be merged in the main product, but these should happen in transifex (like in other apps)

Reply:
We removed this translation file in PR